lynx/src/main/java/com/github/pedrovgs/lynx/LynxShakeDetector.java

```diff
@@ -46,6 +48,9 @@ public void init() {
       @Override public void hearShake() {
         if (isEnabled) {
           openLynxActivity();
+          if (keepAlwaysActive) {
+            disable();
+          }
```

Comment:
This change is not exactly for that. Imagine you start the app with the 'always active' flag enabled. Then you will want a LynxActivity launched if you shake the device even in background, but just one. That's why I've added this check.